simulation.jl

```diff
@@ -84,59 +102,71 @@ end
 function simulate_epochs_data(
     ntime,
     nchannels,
-    events,
-    sampling_rate = 1;
+    events;
+    sampling_rate = 1,
+    coef = shuffle(Vector(0:3)), # define channel weights
```

Comment:
this is a bit confusing to me, why random suffle? Let's discuss this one later

Reply:
also channel weights, arent these the coefficients / betas?

Reply:
> this is a bit confusing to me, why random suffle? Let's discuss this one later

It gives interesting plots when used with `run_simulation( num = 30)`. Though I agree for analysing it is not needed.
> also channel weights, arent these the coefficients / betas?

Late night programming mistakes 😅